### PR TITLE
API-13624 add DeviceRequest

### PR DESCRIPTION
**Pull request by jblefkove-va**

### CODE REVIEW

**Comment by jblefkove-va** [comment]
FYI @haidong @nmoisan @jtaylor-va

**Review by nmoisan — DISMISSED** [review]
LGTM, matches datamart definition

**Review by haidong — DISMISSED** [review]
LGTM!

File: db/migration/V25_0__create_device_request.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE [App].[DeviceRequest]
+(
+    [CDWIdNumber] [BIGINT] NOT NULL,
+    [CDWIdResourceCode] [CHAR](1) NOT NULL,
+    [PatientFullICN] [VARCHAR](50) NOT NULL,
+    [DateUTC] DATETIME NULL,
```
Comment (jtaylor-va) [inline db/migration/V25_0__create_device_request.sql]: Should this be `DATETIME2(0)`? That would match datamart explicitly.

**Review by jtaylor-va — APPROVED** [review]

**Pull request merged**